### DISCUSSION TITLE
Actividad 0 - Morullo, Manuel

### DISCUSSION THREAD

**Discussion by manuel-edlp** [frlp-utn-ingsoft/catedra#28] [Show and tell]
https://github.com/manuel-edlp/Ing-de-Software

**Comment by RodrigoJacznik** [comment]
Hola, la app no funciona. Creo que en la parte 3 cometiste un error en este paso:
![image](https://github.com/frlp-utn-ingsoft/catedra/assets/2715743/d12308ae-4e74-42d2-84f2-df16b4d105fb)

No tenias que reemplazar todo el archivo, solo esa linea. Deberias arreglarlo para aprobar la actividad.
Antes de pushea proba que la app funcione.
Saludos

**Comment by manuel-edlp** [comment]
Hola, ya lo chequeé y lo corregí.

**Comment by RodrigoJacznik** [comment]
Genial. La actividad esta aprobada
